react-nd/src/components/hero/Search.module.scss

```diff
@@ -0,0 +1,56 @@
+.searchBlock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  margin-bottom: 4rem;
+  .input {
+    padding: 1rem 1.5rem;
+    border-radius: 2rem;
+    width: 50%;
+    font-size: 1rem;
+    height: 4rem;
+    &:focus {
+      border: 2px solid #90d81d;
+    }
+  }
+  .btn {
+    background-color: #90d81d;
+    padding: 0.5rem 1rem;
+    border-radius: 50%;
+    width: 4rem;
+    height: 4rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    &:hover {
+      background-color: #84cc16;
+    }
+  }
+  img {
+    width: 1.2rem;
+  }
+
+  // REPONSIVE
+  @media (max-width: 600px) {
```

Comment:
naudokite mixins'us, tam kad išvengti gausos taškų, o būtų naudojami konfiguruojami keli lūžio taškai:
https://dev.to/rembertdesigns/how-to-write-media-queries-with-sass-mixins-32e